sql-plugin/src/main/scala/com/nvidia/spark/rapids/SpillableColumnarBatch.scala

```diff
@@ -479,6 +479,19 @@ class SpillableHostBuffer(handle: SpillableHostBufferHandle,
     handle.materialize()
   }
 
+  /**
+   * Get the host buffer for data part only, which is sliced to the range of [0, length).
```

Comment:
@abellina please take a look at this to confirm this is how you suppose the length field will be used.